src/rust/plugin-registry/src/static_files/hax_docker_plugin.nomad

```diff
@@ -54,6 +54,11 @@ job "grapl-plugin" {
   namespace   = "plugin-${var.plugin_id}"
   type        = "service"
 
+  reschedule {
+    # Make this a one-shot job
+    attempts = 0
+  }
+
```

Comment:
idk if we should just let plugins continue to attempt to come up again forever and ever, or what